sqlmesh/core/context.py

```diff
@@ -1914,7 +1914,9 @@ def _snapshots(
 
         local_nodes = {**(models_override or self._models), **self._standalone_audits}
         nodes = local_nodes.copy()
-        audits = self._audits.copy()
+        audits = {
+            name: audit for name, audit in self._audits.items() if isinstance(audit, ModelAudit)
+        }
```

Comment:
Would it be helpful to add a `_model_audits` attribute to `GenericContext` (like `_standalone_audits`) and use that here instead?

Reply:
Actually, it seems like maybe the type annotation for `_audits` should just be changed to `UniqueKeyDict[str, ModelAudit]`?

Reply:
yea this one is not correct, i think you can just change _audits to be ModelAudit

Reply:
changed